nrf_radio_802154

```diff
@@ -0,0 +1,51 @@
+set(NRF_802154_DRIVER_INCLUDE_DIRS ${NRF_802154_DRIVER_ROOT}/src)
```

Comment:
Is it a cmake file? Should it have `.cmake` suffix or something like this?

Reply:
That file was added by mistake.